### PR TITLE
Add pagerduty to tap test job

### PR DESCRIPTION
**Pull request by taymoork2**

### CODE REVIEW

**Comment by adamweeks** [comment]
Do we need to add `PAGERDUTY_KEY` to the env vars on circle?

**Comment by taymoork2** [comment]
> Do we need to add `PAGERDUTY_KEY` to the env vars on circle?

Correct

File: .circleci/config.yml [hunk 1]
```diff
@@ -350,6 +350,17 @@ jobs:
       - store_artifacts:
           path: /home/circleci/.npm/_logs/
           destination: npm-logs
+      - run:
+          name: 'Create Incident on PagerDuty'
+          when: on_fail
+          command: |
+            curl -X POST --header 'Content-Type: application/json' -d '{
```
Comment (InteractiveTimmy) [inline .circleci/config.yml]: Could / Have we set up a trial run of this CURL request? Would like to confirm before we proceed with merging.

File: .circleci/config.yml [hunk 2]
```diff
@@ -350,6 +350,17 @@ jobs:
...
+              "service_key": "${PAGERDUTY_KEY}",
+              "event_type": "trigger",
+              "description": "FAILURE -- React Widgets Tap Test -- ${CIRCLE_BUILD_URL}",
+              "client": "CircleCI - React Widgets",
+              "client_url": "${CIRCLE_BUILD_URL}"
+            }' https://events.pagerduty.com/generic/2010-04-15/create_event.json
```
Comment (lalli-flores) [inline .circleci/config.yml]: Is there a reason why you didn't use Pager Duty [Event API v2](https://v2.developer.pagerduty.com/docs/send-an-event-events-api-v2)?

File: .circleci/config.yml [hunk 3]
```diff
@@ -350,6 +350,17 @@ jobs:
...
           path: /home/circleci/.npm/_logs/
           destination: npm-logs
+      - run:
+          name: 'Create Incident on PagerDuty'
+          when: on_fail
+          command: |
+            curl -X POST --header 'Content-Type: application/json' -d '{
+              "service_key": "${PAGERDUTY_KEY}",
+              "event_type": "trigger",
+              "description": "FAILURE -- React Widgets Tap Test -- ${CIRCLE_BUILD_URL}",
```
Comment (lalli-flores) [inline .circleci/config.yml]: Should probably add the build URL as part of the details. Having it as part of the summary would make it too crowded/long.

```suggestion
              "description": "FAILURE: React Widgets TAP Test",
```

File: .circleci/config.yml (hunk 2 above)
Comment (taymoork2) [reply .circleci/config.yml]: v2 requires a "from" property which is essentially an email, this just uses the key as authentication which i think its just less setup and cleaner

File: .circleci/config.yml (hunk 3 above)
Comment (taymoork2) [reply .circleci/config.yml]: Yeah that makes sense

File: .circleci/config.yml (hunk 2 above)
Comment (lalli-flores) [reply .circleci/config.yml]: am I missing something? Where is the `from` parameter?
Screen Shot 2020-03-27 at 10 12 30 AM

**Pull request merged**